e2e-pw/src/oss/poms/sidebar.ts

```diff
@@ -0,0 +1,67 @@
+import { Locator, Page } from "src/oss/fixtures";
+
+type filterMode = "select" | "exclude" | "show" | "omit";
+type visibilityMode = "show" | "hide";
+
+export class SidebarPom {
+  readonly page: Page;
+  readonly sidebar: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+
+    this.sidebar = page.getByTestId("sidebar");
+  }
+
+  async clickFieldCheckbox(field: string) {
+    const selector = this.sidebar.getByTestId(`checkbox-${field}`);
+    return selector.click();
+  }
+
+  async clickFieldDropdown(field: string) {
+    const selector = this.sidebar.getByTestId(`sidebar-field-arrow-${field}`);
+    return selector.click();
+  }
+
+  // when less than 25 entries, it's displayed in a list
+  async getAttributeItemCount(field: string, attributeValue: string) {
+    const container = this.sidebar.getByTestId(`categorical-filter-${field}`);
+    const item = container.getByTestId(`checkbox-${attributeValue}`);
+    const countText = item.getByTestId(`entry-count-all`);
+    return countText;
+  }
+
+  // apply a filter to a field
+  async getLabelFromList(
```

Comment:
If we're "apply"ing, can we rename this function to reflect that? I got a little confused when I was reading the specs.